packages/db/src/migrations/0016_autogenerate_user_id.ts

```diff
@@ -8,5 +8,8 @@ export async function up(db: Kysely<any>) {
 }
 
 export async function down(db: Kysely<any>) {
-  await db.schema.alterTable("ow_user").alterColumn("id", (col) => col.setDefault(sql`""`))
+  await db.schema
+    .alterTable("ow_user")
+    .alterColumn("id", (col) => col.setDefault(sql`null`))
```

Comment:
this is just a hotfix for the sql`""` which is invalid

Reply:
You can delete this migration, it's no longer in the main branch